src/app/pages/rpc-sign-bip322-message/rpc-sign-bip322-message.tsx

```diff
@@ -29,13 +30,24 @@ function RpcSignBip322Message() {
     origin,
     message,
     address,
-    isLoading,
+    isLoading: signBip322MessageIsLoading,
     onUserApproveBip322MessageSigningRequest,
     onUserRejectBip322MessageSigningRequest,
   } = useSignBip322Message();
 
+  const location = useLocation();
   const { chain } = useCurrentNetwork();
 
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
This code stops the button from having a loading spinner if you close the dialog and start again.